### PR TITLE
Scav Randomizer V2

### PR DESCRIPTION
**Pull request by ghost**
#### Randomized bots !
the server now generate randomized AI scavs, only based on RNG

##### What is randomized ?
 
- Appearance : head and outfit are mixed with all kind of scavs( bosses, pmcs etcc..)
- weapons : more than 160 possibilities including pistols , grenades & ultra modded weapons
- gear : scavs can have any rigs, armor, backpack and headgear , each slot is based on % of chance
- skills : all their skills are randomized (endurance, strengh, vitality, etcc)

#####  randomization 
Every bot randomized have these following chances of stuff :

- Vest rig : 100% (can be armored, they are mixed with normal rigs)
- Magazines : 100% , randomized magazine on his weapon and two on his rig
- Ammo : 100% : stacks of ammo on each mags are randomized
- Glasses : 30%
- Face cover : 40%
- headwear : 40% (it can be an armored Helmet, but no visor...)
- Backpack : 25%
- Body armor : 25% 
- Grenade in pocket : 10% (except bosses)
- meds in pocket : 10% (except bosses)
- Snipers scavs can only have DMR, carabines, or snipers rifles 

#### Scav bosses ?
Killa and Reshala are vanillia, but reshala's guards stuff is randomized, their behavior are still the same. So be very carefull when hunting bosses

#### How it works ?
the server take care of all !  Nothing to configure, nothing to edit : )

##### PmcWar Mode
the randomizer provide a pmc war mode, it replace randomly every scav by a bear or a usec
witch lead into some epic team fight with AI

therefore, this setting is hard on CPU usage and make the game sluttering ..
there is one option to change in "bots/BotsSettings.json"

    "EnablePmcWar":true,
(its false by default)

________________________________________

PS : I made those change you wanted, easy to tweak if needed
I made a lot of changes but the idea is still the same

in server.JS the only change are two things : the "generateBots" function (  [line103](https://github.com/polivilas/EmuTarkov/pull/12/files#diff-78c12f5adc1848d13b1c6f07055d996eR103) )
and a case on "handleRequest" [line 824](https://github.com/polivilas/EmuTarkov/pull/12/files#diff-78c12f5adc1848d13b1c6f07055d996eR824)...

### CODE REVIEW

**Review by polivilas — COMMENTED** [review]
Is this intentional? I don't think PMC has 70 head health.

File: list.json [hunk 1]
```diff
@@ -721,44 +581,44 @@
 				"BodyParts": {
 					"Head": {
 						"Health": {
-							"Current": 35,
-							"Maximum": 35
+							"Current": 70,
```
Comment (polivilas) [inline list.json]: This is the change in quesiton.

Comment (ghost) [reply list.json]: oops totally forgot to revert normal health for the profile
I will update in a minute : )

**Comment by InNoHurryToCode** [comment]
Ran this version of the server for a couple of hours  (~3-4) on various ingame AI settings without any issues. Didn't test the PMC war through.

**Comment by polivilas** [comment]
:+1:, no bad behavior on my side, thank you!

**Pull request merged**